README.md

```diff
@@ -1,7 +1,79 @@
-# Repo for a potential spring 2023 Votetracker+ demo
+# VoteTrackerPlus Web API
 
-This is a very experimental repository to hold the front and backend code to support a live participatory VoteTracker+ demo, potentially in the spring of 2023.
+A FastAPI interface to the VoteTrackerPlus (VTP) backend, to support a live participatory demo in the spring of 2023.
 
-There is an initial mind map located at [https://www.mindmeister.com/map/2534840002?t=2nMk3h9Uha](https://www.mindmeister.com/map/2534840002?t=2nMk3h9Uha)
+## Useful Links
 
-An initial design document can be found [docs/DesignNotes.md](docs/DesignNotes.md)
+- For an overview of the demo project, check out [the project mind map](https://www.mindmeister.com/map/2534840002?t=2nMk3h9Uha).
+- This repo also includes the official [Design Notes](docs/DesignNotes.md).
+- The API endpoints in this project are a web interface to the [VoteTrackerPlus backend](https://github.com/TrustTheVote-Project/VoteTrackerPlus).
+
+## Installation
+
+This Python project uses [poetry](https://python-poetry.org/) for dependency and package management. To run the code in this repo, first [install poetry](https://python-poetry.org/docs/#installation) on your development workstation. Then,
+
+1. Clone or copy the code to a directory where you keep your GitHub repositories.
+2. Enter the `VTP-web-api` directory you just created, like this: `cd ~/repos/VTP-web-api`
+3. To install the required Python packages, run `poetry install`
+4. To use the new virtual environment you just created, and run the API server & tests (see below for details), run `poetry shell`
+
+Note: if you want to use a certain Python version, you can tell poetry, like this:
+
+```bash
+poetry env use 3.9
+```
+
+This command will set up a virtual environment using Python 3.9. Note that the specified Python version must already be installed on your computer.
+
+This project requires Python 3.9 or later.
+
+## Run the API server
+
+1. Once the installation is complete, go to the source code directory: `cd src/vtp/web/api`
+2. Run the `uvicorn` server like this: `uvicorn main:api`
+
```

Comment:
FYI - when I get to this step, it errors for me:

(vtpwebapi-py3.10) [sandy] VTP-web-api % cd src/vtp/web/api                                                                    
(vtpwebapi-py3.10) [sandy] api % uvicorn main:api                                                                              
ERROR:    Error loading ASGI app. Attribute "api" not found in module "main".

Still looking into it but wanted you to know.

Reply:
Oops, my bad! It's a typo on my part, the correct command is:

```bash
uvicorn main:app
```

(Not `main:api` as I typed).

I'll correct the file now. Thanks!